src/pages/index.js

```diff
@@ -11,17 +11,19 @@ import { retrieveUserToken } from "../util/auth"
 
 import "./index.css"
 
+import "semantic-ui-css/semantic.min.css"
```

Comment:
Have you made sure that this does not mess up the existing styling? Looks like the login page was messed up.

You could also look into [this](https://www.gatsbyjs.com/plugins/gatsby-plugin-styled-components/) to make sure that there are no classname collisions with classes generated with styled-components. But I suspect the login page can be fixed by this.